### PR TITLE
Get / Create project commands

### PR DESCRIPTION
**Pull request by febg**
## Create
- Added command description
- Replaced `jsonFlag` for `fmtFlag`
- Added success message upon successful project creation

## Get
Finds a project a user is member of based on project id
**TODO:** Add find by name functionality

### Flags
- id (required)

### Example
```
-> ./padl project get --id b18031e1-3081-447e-bd6f-18a6a144afe6

{
    "ID": "b18031e1-3081-447e-bd6f-18a6a144afe6",
    "Name": "felipepp22",
    "Members": {
        "felipeballesteros1@gmail.com": 2
    },
    "DeployKeys": {},
    "PadlfileHash": "828c57caf665d7a993dfbc2d3ea8a256babfd227a2b521178c2bf01da34f2c294141a6c7cae2fa96f1f935765c67711fd9ce493a256c7a5c3899e656bce4a66a"
}
```

### CODE REVIEW

File: api/client/project.go [hunk 1]
```diff
@@ -52,3 +54,28 @@ func (p *Padl) CreateProject(name, description string) (*padlfile.File, error) {
 
 	return &pf, nil
 }
+
+func (p *Padl) GetProject(pid string) (*project.Project, error) {
+	req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("%s/project/%s", p.HostURL, pid), nil)
+	if err != nil {
```
Comment (adrianosela) [inline api/client/project.go:60]: You gotta get pid from request context

File: cli/commands/project.go [hunk 2]
```diff
@@ -76,5 +87,31 @@ func createProjectHandler(ctx *cli.Context) error {
...
+
+	prettyJSON, err := json.MarshalIndent(project, "", "    ")
+	if err != nil {
+		return fmt.Errorf("Failed to generate json: %s", err)
+	}
+	fmt.Printf("%s\n", string(prettyJSON))
+	return nil
+}
+
+func getProjectValidator(ctx *cli.Context) error {
```
Comment (adrianosela) [inline cli/commands/project.go]: Move validator above habdlers as in kms.go

**Review by adrianosela — APPROVED** [review]
Nits. Fix and merge

File: api/client/project.go (hunk 1 above)
Comment (febg) [reply api/client/project.go:60]: @adrianosela not clear what you mean.

Comment (adrianosela) [reply api/client/project.go]: See getting keyd id in kms.go

Comment (adrianosela) [reply api/client/project.go:60]: My bad im tripping. Ignore

**Pull request merged**